src/LessonDao.java

```diff
@@ -17,14 +17,14 @@ public Lesson insert(Lesson entity) {
         if (entity.getId() != null) {
             throw new JdbcOperationException("Id must not be provided during the insert operation!");
         }
-        var sql = """
+        final var SQL = """
```

Comment:
This is not a constant! Constant should be declared on the class level! Like this: 
`private static final String INSERT_SQL_QUERY = "INSERT INTO schedule.lesson(name, homework_id) VALUES (?, ?)";`. Do the same for all other queries in LessonDao